examples/classic/distributed/ewd840/ewd840.qnt

```diff
@@ -115,109 +104,100 @@ action Deactivate(i) = all {
-(***************************************************************************/
+/// Liveness property: termination is eventually detected.
 temporal Liveness =
-  always(Node.forall(i => not(active.get(i))) implies eventually(terminationDetected))
+  always(Nodes.forall(i => not(active.get(i))) implies eventually(terminationDetected))
 
-/***************************************************************************)
-(* The following property asserts that when every process always           *)
-(* eventually terminates then eventually termination will be detected.     *)
-(* It does not hold since processes can wake up each other.                *)
-(***************************************************************************/
+/// The following property asserts that when every process always eventually
+/// terminates then eventually termination will be detected. It does not hold
+/// since processes can wake up each other.
 temporal FalseLiveness =
-  always(Node.forall(i => always(eventually(not(active.get(i)))))
+  always(Nodes.forall(i => always(eventually(not(active.get(i)))))
          implies eventually(terminationDetected))
 
-/***************************************************************************)
-(* The following property says that eventually all nodes will terminate    *)
-(* assuming that from some point onwards no messages are sent. It is       *)
-(* not supposed to hold: any node may indefinitely perform local           *)
-(* computations. However, this property is verified if the fairness        *)
-(* condition WF_vars(Next) is used instead of only WF_vars(System) that    *)
-(* requires fairness of the actions controlled by termination detection.   *)
-(***************************************************************************/
-temporal SpecWFNext = Init and always(Next.orKeep(vars) and Next.weakFair(vars))
-
-temporal AllNodesTerminateIfNoMessages = (
-  eventually(always(
-      Node.forall(i => not(SendMsg(i))).orKeep(vars)
-  )) implies eventually(Node.forall(i => not(active.get(i))))
-)
-
-/***************************************************************************)
-(* Dijkstra's inductive invariant                                          *)
-(***************************************************************************/
+/// The following property says that eventually all nodes will terminate
+/// assuming that from some point onwards no messages are sent. It is not
+/// supposed to hold: any node may indefinitely perform local computations.
+/// However, this property is verified if the fairness condition WF_vars(step)
+/// is used instead of only WF_vars(System) that requires fairness of the
+/// actions controlled by termination detection.
+///
+/// Quint translation note: we cannot add `init` to this property since, in quint,
+/// it is an action - not a predicate
+temporal SpecWFstep = always(step.orKeep(vars) and step.weakFair(vars)) // and init
```

Comment:
I guess we should check with @konnov on how to treat `init` in temporal formulas, and note this here with a reference to an issue.

Reply:
@konnov pinging you here since you probably missed it!

Reply:
ah, I think it is the fifth time I am staring at it. Let me open an issue.

Reply:
Let's maybe refer to the issue and comment out the whole formula?

Reply:
Thanks! Ok, will do